### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Properties_Blob Container Propertiestab_NonTextContrast: In System Default/Light theme Color Contrast of focus border on properties "Action" button failing minimum Color Contrast ratio is 3:1

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6321]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**

#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Dec2022; #Azure Tools Storage Explorer; #WCAG1.4.11; #FTP; #A11yWCAG2.1; #ColorContrast; #NonTextContrast; 
### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
### OS Details
Microsoft Windows 11 Enterprise
version 22H2 OS Build 22621.819
### Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign into Azure.
3. Go to "Manage Accounts" pane and select the subscriptions you will use. Once selected, click on "Apply" button.
4. Select "Blob Containers" of the selected subscription.
5. Navigate to "Properties" Tab present on left bottom of the screen and hit enter. 
6. Use Color Contrast Tool to check the color contrast of keyboard focus border on properties "Action" button is less than 3:1 or not.

### Actual results:
In System Default/Light theme Color Contrast of keyboard focus border on properties "Action" button failing minimum Color Contrast ratio is 3:1 (given 2.4:1)

**Similar issues observed in the following:**

- In System Default/Light theme,

1. Click on any Blob container, Toolbar menu is visible. All the tools present in the above table is failing color contrast for visible focus.
2. Click on any Disk, Toolbar menu is visible. All the tools present in the above table is failing color contrast for visible focus.
3. Click on any Blob container. Tab to properties present at the left bottom is failing color contrast of properties border is less than 3:1
4. Click on any blob container and right to select Get shared access signature. Tab to calendar button is failing color contras for focus visible.
5. Click on any blob container and navigate to Activities button and activities action dropdown present at the right bottom color contrast is failing
### Expected results:
Color contrast of keyboard visible focus should have a minimum of 3:1 ratio when goes is on properties “Action” button.
### User Impact:
Low vision Users might fail to acknowledge the focused element if it doesn't have minimum color contrast ratio.
### Attachments:
![Action dropdown](https://user-images.githubusercontent.com/100552367/205271036-3b3fd8dc-e038-4630-9394-2c3caf95c164.png)
![Tool Bar](https://user-images.githubusercontent.com/100552367/205271066-8e0e3eb5-4624-4a49-be63-3930788260d1.png)
![Properties](https://user-images.githubusercontent.com/100552367/205271088-3d160c74-674e-4c2d-9ab8-9c76b4fd5b91.png)
![Activities](https://user-images.githubusercontent.com/100552367/205271612-5701a38f-526a-4ead-a523-052abcf123e4.png)

**Comment by Jtith** [comment]
cralvord@microsoft.com    verified in the latest build 1.28.0 issue is fixed partially, still repro on tool bar and calendar control keyboard focus is less than 3:1
![6321_still repro](https://user-images.githubusercontent.com/100552367/211463832-d98bd580-a899-4117-8072-d4ab02ba5128.png)

**Comment by craxal** [comment]
@Jtith Can you please open the calendar control in a separate issue? It's completely separate from the original issue.

**Comment by Jtith** [comment]
Issue is not repro'ing calendar filed is getting expand by pressing arrow key, hence closing the bug.
